pages/projects/life-journeys.js

```diff
@@ -0,0 +1,136 @@
+        <meta
+          name="dcterms.language"
+          content={props.locale === "en" ? "eng" : "fra"}
+        />
+        <meta name="dcterms.creator" content={t("creator")} />
+        <meta name="dcterms.accessRights" content="2" />
+        <meta name="dcterms.service" content="ESDC-EDSC_SCLabs-LaboratoireSC" />
+      </Head>
+      <section className="layout-container relative mb-10">
+        <h1
+          id="pageMainTitle"
+          className="mb-8 text-h1l font-bold flex-wrap"
+          tabIndex="-1"
+        >
+          {t("lj:lifeJourneysTitle")}
+        </h1>
+        <h2 className="mb-6 font-bold leading-10">
+          {t("lj:lifeJourneysHeading1")}
+        </h2>
+        <p className="mb-6">{t("lj:lifeJourneysContent1")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent2")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent3")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent4")}</p>
+        <h2 className="mb-6 font-bold leading-10">
+          {t("lj:lifeJourneysHeading2")}
+        </h2>
+        <figure className="mb-6 lg:w-2/3 border shadow-experiment-shadow">
+          <img
+            src={t("lj:lifeJourneysImg1")}
+            alt={t("lj:lifeJourneysImgAltText1")}
+          ></img>
+          <figcaption className="p-2.5 text-sm font-display border-t">
+            {t("lj:lifeJourneysImgCaption1")}
+          </figcaption>
+        </figure>
+        <p className="mb-6">{t("lj:lifeJourneysContent5")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent6")}</p>
+        <p>{t("lj:lifeJourneysContent7")}</p>
+        <HTMList
+          tag="ol"
+          listClassName={"ml-9 mb-6 text-p list-disc"}
+          content={t("lj:lifeJourneysListItems1")}
+        />
+        <h2 className="mb-6 font-bold leading-10">
+          {t("lj:lifeJourneysHeading3")}
+        </h2>
+        <figure className="mb-6 lg:w-2/3 border shadow-experiment-shadow">
+          <img
+            src="/life-journey-map.png"
+            alt={t("lj:lifeJourneysImgAltText2")}
+          ></img>
+          <figcaption className="p-2.5 md:text-center text-sm font-display border-t">
+            {t("lj:lifeJourneysImgCaption2")}
+          </figcaption>
+        </figure>
+        <p className="mb-6">{t("lj:lifeJourneysContent8")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent9")}</p>
+        <p>{t("lj:lifeJourneysContent10")}</p>
+        <HTMList
+          tag="ol"
+          listClassName={"ml-9 mb-6 text-p list-disc"}
+          content={t("lj:lifeJourneysListItems2")}
+        />
+        <p className="mb-6">{t("lj:lifeJourneysContent11")}</p>
+        <h2 className="mb-6 font-bold leading-10">
+          {t("lj:lifeJourneysHeading4")}
+        </h2>
+        <p className="mb-6">{t("lj:lifeJourneysContent12")}</p>
+        <p className="mb-6">{t("lj:lifeJourneysContent13")}</p>
+      </section>
+
+      <CallToAction
```

Comment:
This `CallToAction` is outdated, it should be:
![image](https://user-images.githubusercontent.com/77116011/126333078-be5477d2-8c12-465a-bab0-edb35c7a76a6.png)

Reply:
Done!